### ISSUE TITLE
Action bar styling issue

### ISSUE THREAD

**Issue by git-seb** [GonzaArts/restore_order_magento2#1]
The action bar has a styling issue, when clicking the dropdown, you have to scroll down inside the action bar.

![image](https://github.com/user-attachments/assets/1d0e3d4c-1131-4a08-a38b-50560b7af6d1)

This is caused by class actions-scrollable in the action bar div alongside page-main-actions. Solution is getting rid of actions-scrollable a.k.a. making the action bar non-scrollable.

**Comment by GonzaArts** [comment]
Reviewing your problem I wanted to see what happened to solve it but as you can see in my magento is going correctly, it may be that you are using a theme in the backend that makes this modification?

![image](https://github.com/user-attachments/assets/2a87ba6f-6a70-4cb8-86ae-0a7b95dce5eb)

**Comment by GonzaArts** [comment]
If you need me to check it in your topic let me know so I can check it.

I will close the issue

**Comment by git-seb** [comment]
Thanks for reviewing. I digged more into this. We are actually using a plugin to change back-end admin colors. But after disabling, still having the same issue. After digging even further, I found that this is actually core magento behaviour, at least in my Magento 2.4.7-p3. I checked Magento 2's github and looks like they have added it here: https://github.com/magento/magento2/commit/6ed183273dd5bc9ae48c73dc9bfc27122b9c8bee (see line 28); so maybe they are still working on it since they added a new line in the _actions-bar.less file "ToDo UI -> Consist old styles, should be changed with new design".
